src/test/java/com/xceptance/xlt/engine/resultbrowser/PageTransformerTest.java

```diff
@@ -278,6 +279,31 @@ public void testTransformLWPage_BaseUrlHandling() throws Throwable
         }
     }
 
+    @Test
+    public void testTransformHtmlPage_MetaRefreshRemoved() throws Throwable
+    {
+        try (final WebClient wc = new WebClient())
+        {
+            final MockWebConnection conn = new MockWebConnection();
+            conn.setDefaultResponse("<html><head><meta http-equiv=\"content-type\" content=\"text/html; charset=UTF-8\"><meta http-equiv=\"content-security-policy\" content=\"default-src 'self'\"><meta http-equiv=\"refresh\" content=\"10; url=/\"></head></html>");
+            wc.setWebConnection(conn);
+
+            final HtmlPage page = wc.getPage(url);
+            
+            final PageDOMClone clone = DomUtils.clonePage(page);
+            final PageTransformer instance = new PageTransformer(clone, true);
+            final UrlMapping urlMapping = new DumpMgr().getUrlMapping();
+
+            final Document doc = instance.transform(urlMapping);
+
+            // check that the refresh http-equiv meta tag has been removed, but not the other two http-equiv metas
+            final NodeList metas = doc.getElementsByTagName("meta");
+            Assert.assertEquals(2, metas.getLength());
+            Assert.assertEquals("content-type", ((Element) metas.item(0)).getAttribute("http-equiv"));
```

Comment:
If I'm not mistaken, `PageTransformer` should remove all meta tags whose `http-equiv` attribute has value `refresh` or `content-type`. Thus, only one meta tag should remain after transformation.

Reply:
Should have read the entire method body; there is a meta tag added later on.